### PR TITLE
Feature/schema module

### LINKED ISSUE #352: Rescaffold objects -> Schema module
Rescaffold the module,  needs a genesis object with a `SchemaMap`.

Dependencies from cosmos include:

* Capability
* Bank

REF: *https://docs.cosmos.network/main/modules/*

Acceptance Criteria

* Remove the Object module
* Add Schema module
* Add Schema type struct
* Add CRUD request type structs

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202528620406598/1202534190425162) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by mcjcloud**
Closes #352 

Delete the `object` module and scaffold a new `schema` module for ADR-002.

## Changes

- Removed object module
- Scaffolded schema module
- Created but didn't implement CRUD operations
- Commented out references to object module in legacy channel/bucket modules.
- Made simulation run (bucket and channels fail still)

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Comment by ntindle** [comment]
```
Error: internal/highway/highway.go:36:38: s.CreateObject undefined (type *api.HighwayServer has no field or method CreateObject)
Error: internal/highway/highway.go:37:38: s.UpdateObject undefined (type *api.HighwayServer has no field or method UpdateObject)
Error: internal/highway/highway.go:38:42: s.DeactivateObject undefined (type *api.HighwayServer has no field or method DeactivateObject)
Error: The process '/Users/runner/hostedtoolcache/goreleaser-action/1.9.2-pro/x64/goreleaser' failed with exit code 1
0s
```

File: proto/schema/v1/tx.proto [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+message MsgCreateSchemaResponse {
+  Schema schema = 1;
+}
+
+message MsgDeprecateSchema {
+  string creator = 1;
+  string did = 2;
+}
+
+message MsgDeprecateSchemaResponse {}
```
Comment (joshLong145) [inline proto/schema/v1/tx.proto]: Should we provide a status in this response? an empty message type feels a bit weird to me.

**Review by joshLong145 — COMMENTED** [review]
Looking good, just one question on the msg return definition for deactivate schema

File: proto/schema/v1/tx.proto (hunk 1 above)
Comment (mcjcloud) [reply proto/schema/v1/tx.proto]: Yes good point. I updated both of the response models to match the other modules (added code and message fields).

**Review by joshLong145 — APPROVED** [review]
LGTM

**Pull request merged**